message SummonPool
{
	optional int32	id 			= 1;		//卡池id
	optional int32	free_used	= 2;		//今日已免费召唤次数
	optional string	next_free	= 3;		//下次免费召唤时间（int64时间戳）
}

message DseSummonInterface
{
	repeated SummonPool 	pools = 1;
}